### PR TITLE
Adding new event to case service

### PR DESCRIPTION
**Pull request by creativeappdevamit**
# What and why?

# How to test?

# Trello
https://trello.com/c/7VEdT3IE/910-allow-change-to-not-started-for-a-submission-in-response-operations-ui

### CODE REVIEW

File: src/main/resources/database/changes/release-33/add_category_entry.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+INSERT INTO casesvc.category
+VALUES ('FORCED_COMPLETED_TO_NOTSTARTED', 'Forced changed from completed to not started', 'Forced changed to not started', 'ACTIVATED', NULL, NULL, NULL, 'B, BI', NULL, NULL)
```
Comment (insacuri) [inline src/main/resources/database/changes/release-33/add_category_entry.sql:2]: The capitalised status and the description aren't written in great english and should be changed.  Not sure off the top of my head what it should be... if anyone has any ideas before I get some time to think about it then mention it!

Comment (creativeappdevamit) [reply src/main/resources/database/changes/release-33/add_category_entry.sql:2]: you might be right! Not great in that :-)

File: src/main/java/uk/gov/ons/ctp/response/casesvc/state/CaseSvcStateTransitionManagerFactory.java [hunk 2]
```diff
@@ -235,6 +237,17 @@
         CategoryDTO.CategoryName.NO_LONGER_REQUIRED,
         CaseGroupStatus.NOLONGERREQUIRED);
 
+    // New transition to enable users to change completedbyphon and no longer required to not
```
Comment (insacuri) [inline src/main/java/uk/gov/ons/ctp/response/casesvc/state/CaseSvcStateTransitionManagerFactory.java:240]: Just need to change `completedbyphon` to `completed by phone`

**Review by insacuri — APPROVED** [review]

**Pull request merged**